fe/fe-core/src/test/java/com/starrocks/sql/plan/JoinTest.java

```diff
@@ -2072,7 +2067,7 @@ public void testEquivalenceLoopDependency() throws Exception {
     public void testJoinCastFloat() throws Exception {
```

Comment:
should also add a new test case to cover join on cast float logic.

Reply:
done